README.md

```diff
@@ -190,15 +190,20 @@ this larger namespace.  But, we don't want to be so general that the spec become
    All attributes are optional but recommended for disambiguation and good meta-data for apps.
 
 
-- **\<podcast:guest href="[url of bio/wiki/blog/etc.]" img="[link to image/headshot]">**[name of person]**\</podcast:guest>**
+**\<podcast:persons/>**
 
-   Item
+   Channel or Item (optional | single)
 
-   (optional | multiple)
+   This element lists optional `<podcast:person/>` item elements that reflects persons of interest for the show.
 
-   It identifies a guest in a podcast episode.
+- **\<podcast:person name="[(string)]" role="[(string)]" img="[(uri of content)]" href="[(uri to website/wiki/blog)]"/>**
```

Comment:
Would this eliminate the need for the `<podcast:host>` tag? Instead it would just be a `role="host"`?

Reply:
Ah, yes. It would solve that as well. I will modify the PR.